lms/templates/dashboard/_dashboard_credit_info.html

```diff
@@ -0,0 +1,70 @@
+<%page args="credit_status" />
+<%!
+    import datetime
+    import pytz
+    from django.utils.translation import ugettext as _
+    from util.date_utils import get_default_time_display
+%>
+<%namespace name='static' file='../static_content.html'/>
+
+% if credit_status["eligible"]:
+    <div class="message message-status is-shown credit-message">
+    % if credit_status["error"]:
+        <p class="message-copy credit-error-msg">
+            ${_(
+                "An unexpected error occurred while retrieving your credit status.  "
+                "Please contact {support_email}."
+            ).format(
+                support_email='<a href="mailto:{address}">{address}</a>'.format(
+                    address=settings.DEFAULT_FEEDBACK_EMAIL
+                )
+            )}
+        </p>
+    % elif not credit_status["purchased"]:
+        <p class="credit-eligibility-msg">
+            % if credit_status["deadline"] < datetime.datetime.now(pytz.UTC) + datetime.timedelta(days=30):
+                ${_("{username}, your eligibility for credit expires on {deadline}. Don't miss out!").format(
```

Comment:
The opportunity to purchase credit for this course expires on Jul 20, 2015 at 21:32 UTC. You’ve worked hard - don’t miss out!

(Is this OK? Does Marketing need to look at it?)

Note that I've omitted the names in this and the next text strings. I always feel vaguely disconcerted when a program/company addresses me in such a familiar way. But maybe that's just me?